app/services/inherited_proofing/va/service.rb

```diff
@@ -0,0 +1,67 @@
+require_relative 'service_payload_validator'
+
+module InheritedProofing
+  module Va
+    # Encapsulates request, response, error handling, validation, etc. for calling
+    # the VA service to gain PII for a particular user that will be subsequently
+    # used to proof the user using inherited proofing.
+    class Service
+      REQUEST_URI = 'https://staging-api.va.gov/inherited_proofing/user_attributes'.freeze
```

Comment:
I feel moderately strongly we should move the domain part of this to be a config value, probably easier to manage if it's not a constant

Reply:
Thank you @zachmargolis this is something @holytoastr suggested and I agree.

Reply:
Done, ty for the feedback.